src/mlpack/methods/det/dtree_impl.hpp

```diff
@@ -1036,3 +1033,6 @@ void DTree<MatType, TagType>::serialize(Archive& ar,
       FillMinMax(minVals, maxVals);
   }
 }
+
+} // namespace det
+} // namespace mlpack
```

Comment:
I think this needs a third closing brace (for the `details` namespace), but I'm not sure if that will fix the build.

Reply:
it is already closed at 148

Reply:
Oh, sorry!  I overlooked that.